databricks_cli/configure/config.py

```diff
@@ -28,54 +28,39 @@
 
 from databricks_cli.configure.provider import DEFAULT_SECTION, get_config_for_profile
 from databricks_cli.utils import error_and_quit
-from databricks_cli.sdk import ApiClient, DbfsService, WorkspaceService, JobsService, \
-    ClusterService, ManagedLibraryService
+from databricks_cli.sdk import ApiClient
 
 
 def require_config(function):
+    """
+    All callbacks wrapped by require_config expect the argument ``profile`` to be passed in.
+    """
     @six.wraps(function)
     def decorator(*args, **kwargs):
-        config = get_config_for_profile(DEFAULT_SECTION)
+        profile = kwargs.pop('profile')
+        config = get_config_for_profile(profile)
         if not config.is_valid:
-            error_and_quit(('You haven\'t configured the CLI yet! '
-                            'Please configure by entering `{} configure`').format(sys.argv[0]))
+            if profile == DEFAULT_SECTION:
```

Comment:
We should move this section out as separate methods since what we need here is simply get a config and assert valid.